shared/actions/chat2/index.tsx

```diff
@@ -2534,6 +2534,27 @@ const joinConversation = async (_: TypedState, action: Chat2Gen.JoinConversation
   )
 }
 
+const fetchUserBio = async (state: TypedState, action: Chat2Gen.SelectConversationPayload) => {
+  const {conversationIDKey} = action.payload
+  const meta = Constants.getMeta(state, conversationIDKey)
+  const otherParticipants = Constants.getRowParticipants(meta, state.config.username || '').toArray()
```

Comment:
we're moving away from immutable.js soon but in the meantime
1. you can just use the list and not convert to an array